modules/account-lib/test/unit/coin/eth2/keyPair.ts

```diff
@@ -0,0 +1,81 @@
+import should from 'should';
+import * as BLS from '@chainsafe/bls';
+import { KeyPair } from '../../../../src/coin/eth2';
+import * as testData from '../../../resources/eth2/eth2';
+
+const pub = testData.ACCOUNT_1.publicKey;
+const prv = testData.ACCOUNT_1.privateKey;
+
+before(async function f() {
+  await BLS.initBLS();
```

Comment:
oh, you should do this inside account-lib - we don't want to make users always do this before using account-lib